src/components/search/SearchResults/SearchResultsCard.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import 'antd/dist/antd.css';
 import { Card } from 'antd';
+import { Rate } from 'antd';
```

Comment:
Do these antd components always import this way, or is grouping them as {Card, Rate} not readable?

Reply:
These can also be grouped like: import { Card, Rate } from 'and';